transmart-core-db/grails-app/services/org/transmartproject/db/querytool/PatientSetQueryBuilderService.groovy

```diff
@@ -321,4 +328,43 @@ class PatientSetQueryBuilderService {
 
         spec.postProcessQuery "$spec.factTableColumn IN ($res)", userInContext
     }
+
+    /**
+     * Build query to retrieve patient counts from table with precalculated counts.
+     *
+     * @param OntologyTerm term
+     * @return sql query as a String
+     */
+    public String buildPatientCountQuery(OntologyTerm term) {
```

Comment:
Who needs this?

Reply:
OntologyTermSerializationHelper#convertToMap
It seems to be used in v1 api. 
cc @wardweistra 

Should we remove it or use new logic for patient counts instead?